hw3/AilurusUmbra/performance.txt

```diff
@@ -0,0 +1,4 @@
+in_test
+Start multiply_naive (repeat=5), take avg = 0.04350147247314453 seconds
+Start multiply_mkl (repeat=5), take avg = 0.0054852962493896484 seconds
+MKL speed-up over naive: 7.93056 x
```

Comment:
this is very weird, cuz it is usually more than 40x. You may want to figure out the reasons.

Reply:
Sorry for that I use too small matrices which are not larger than 500x500. I've increased the size in timing code and added new `performance.txt` to this PR.